### DISCUSSION TITLE
How to include standalone Nixvim flake to Home-Manager flake?

### DISCUSSION THREAD

**Discussion by jdsee** [nix-community/nixvim#1240] [Q&A] [answered]
Hi, I set up a standalone Nixvim flake as described in the docs. Now I would like to include this flake into my flake based HM configuration.

I added my Nixvim flake to the inputs of my HM flake but don't understand how to include the neovim package.

Has anyone done this or an idea how to solve that?

Here are the flakes, in case that helps:
- HM: https://github.com/jdsee/nix-config
- Nixivm: https://github.com/jdsee/nixvim-config

**Comment by siph** [comment, marked as answer]
You just need to put the derivation into `home.packages`:
```nix
{ inputs, system, ... }:
{
  # NixOS
  environment.systemPackages = [ inputs.nixvim-config.packages.${system}.default ];
  # home-manager
  home.packages = [ inputs.nixvim-config.packages.${system}.default ];
}
```

More here: https://gist.github.com/siph/288b7c6b5f68a1902d28aebc95fde4c5

**Reply by jdsee** [reply]
Great, thanks a lot!

I think your Gist is really helpful. Would be great, if it was referenced in the docs.

**Reply by traxys** [reply]
Added the snippet in the doc!
